src/components/shared/ArrowPanel.css

```diff
@@ -165,8 +165,6 @@
   font-size: 12px;
   line-height: 20px;
   padding-top: 5px;
-  max-height: calc(100vh - 60px);
-  overflow-y: auto;
```

Comment:
This is taken care of by the rules above for `.arrowPanelContent`, and `.arrowPanelcontent` is the better place for this. Using `overflow` for `arrowPanel` like here hides the arrow.